shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -159,6 +160,10 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         result.setSelected((object.flags & SEMANTICS_FLAG_IS_SELECTED) != 0);
         result.setText(object.label);
 
+        if ((object.flags & SEMANTICS_FLAG_IS_BUTTON) != 0) {
```

Comment:
Maybe also modify the iOS implementation to match this?

https://github.com/flutter/engine/blob/master/shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm#L167

Reply:
I'll follow up with a separate CL for iOS (need to switch machines for that :smirk: )